src/edu/csus/ecs/pc2/core/scoring/DefaultScoringAlgorithm.java

```diff
@@ -245,6 +247,72 @@ private ProblemSummaryInfo calcProblemScoreData(TreeMap<Run,Run> treeMap, IInter
         return problemSummaryInfo;
     }
 
+    /**
+     * Get the Point Scoring and Statistics information for one problem.
+     *
+     * @return pc2.ex.ProblemScoreData
+     * @param treeMap
+     *            java.util.TreeMap
```

Comment:
It would be nice if the `@param` explained what the `treeMap` is expected to contain...

Reply:
Haha.  I added that when I fixed the comment.
Next push.